### PR TITLE
Base: Add Danish keymap

### PR DESCRIPTION
**Pull request by RasmusNylander**
Based on what characters Windows 10 wrote when a key was pressed.

This makes it a lot nicer to use a Danish keyboard :^)

### CODE REVIEW

File: Base/res/keymaps/da.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "map":       [ "", "\u001B", "1",  "2", "3", "4", "5", "6", "7", "8", "9", "0", "+", "´", "\b",   "", "q", "w", "e", "r", "t", "y", "u", "i", "o", "p", "å", "¨", "\n", "", "a", "s", "d", "f", "g", "h", "j", "k", "l", "æ", "ø", "½", "", "'", "z", "x", "c", "v", "b", "n", "m", ",", ".", "-", "", "*", "", " ", "", "", "", "", "", "", "", "", "", "", "", "", "", "7", "8", "9", "-", "4", "5", "6", "+", "1", "2", "3", "0", ",", "", "",  "<", "", "", ""],
```
Comment (oliverhaagh) [inline Base/res/keymaps/da.json:2]: Shouldn't it be "\t" before "q" instead of ""?

Comment (RasmusNylander) [reply Base/res/keymaps/da.json:2]: Yes, indeed it should! I seem to have swapped _map_ and _shift_map_ here; I'll get on it right away (or perhaps after I've eaten dinner)

Comment (RasmusNylander) [reply Base/res/keymaps/da.json]: That should do it!

Comment (oliverhaagh) [reply Base/res/keymaps/da.json]: Looks good to me! BTW very cool that you took the time to make this! :^)

Comment (RasmusNylander) [reply Base/res/keymaps/da.json]: Thanks! :^D

**Pull request merged**